javascript-bank/bank.js

```diff
@@ -0,0 +1,49 @@
+/* exported Bank */
+
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (balance <= 0 || Number.isInteger(balance) === false) {
+    return null;
+  } else {
+    var newAccount = new Account(this.nextAccountNumber, holder);
+    newAccount.deposit(balance);
+    this.accounts.push(newAccount);
+    this.nextAccountNumber += 1;
+    return newAccount.number;
+
+  }
+
+};
+
+Bank.prototype.getAccount = function (number) {
+
+  if (number >= this.nextAccountNumber) {
+    return null;
+  } else {
+    return this.accounts[number - 1];
+
+  }
+
+};
```

Comment:
`getAccount` should always return `null` if the `account.number` equal to `number` doesn't exist.

---

![image](https://user-images.githubusercontent.com/16143771/150912761-94cfe3bf-7669-4bde-90c1-2dd21127aecb.png)